packages/aquedux-client/src/network/client.js

```diff
@@ -193,9 +187,8 @@ const createAqueduxClient = (store, why, config) => {
   }
 
   return {
-    start: endpoint => {
-      store.dispatch(actions.config.setEndpoint(endpoint))
-      start(endpoint)
+    start() {
+      start()
```

Comment:
I like the fact that the endpoint is now in the configuration and no more an argument of the start method. 👍